src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -71,17 +74,41 @@ public void init() throws ServletException {
   // TODO: Check if URL is valid.
```

Comment:
Can this be removed yet?

Reply:
IIRC This TODO was to check if an actual youtube video exists with the provided URL

Reply:
Can you rephrase this TODO to be more descriptive?

Reply:
Done, also added tracking issue #224